### PR TITLE
Improve Vercel GraphQL example

### PR DESCRIPTION
**Pull request by 2color**
## What
- Upgrade `nexus-prisma` -> nexus-plugin-prisma/schema
- Make fields required in queries
- Add users queries
- Add better dev command

### CODE REVIEW

File: deployment-platforms/vercel-graphql/api/_lib/context.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { PrismaClient } from '@prisma/client'
 
-const prisma = new PrismaClient()
+const prisma = new PrismaClient({
+  log: ['query', 'info', 'warn'],
```
Comment (divyenduz) [inline deployment-platforms/vercel-graphql/api/_lib/context.ts:4]: Do we want to keep this logging in the examples?

Comment (2color) [reply deployment-platforms/vercel-graphql/api/_lib/context.ts:4]: Yes.

Comment (2color) [reply deployment-platforms/vercel-graphql/api/_lib/context.ts:4]: My thought was that this provides useful insight into what's happening with the application as queries come in.

Comment (Jolg42) [reply deployment-platforms/vercel-graphql/api/_lib/context.ts:4]: I would put it commented with a comment "Useful for debugging..." or remove it personally

**Pull request merged**